jenkins_jobs/trigger_nightly_host_os_build/pre_build_script.sh

```diff
@@ -1,2 +1,15 @@
 RELEASE_DATE=$(date +%Y-%m-%d)
+COMMIT_BRANCH="nightly-${RELEASE_DATE}"
+
+# Upgrade versions
+# sudo yum install rpmdevtools
+python host_os.py \
+   --verbose \
+   upgrade-versions \
+       --committer-name "$GITHUB_BOT_NAME" \
+       --committer-email "$GITHUB_BOT_EMAIL" \
+       --push-repo-url "ssh://git@github.com/${GITHUB_BOT_USER_NAME}/versions.git" \
+       --push-repo-branch "$COMMIT_BRANCH"
```

Comment:
I've created a PR for upgrade-versions modification required. After reviews I'll update this one.